### ISSUE TITLE
Android: Stuck Installing APK

### ISSUE THREAD

**Issue by SPENCRY** [koush/vysor.io#881]
**To Reproduce**
The main device screen shows the Android phone connected.
Click the Play Button.
Vysor asks to allow USB debugging on the phone.
I allow it (I think, the screen is broken and barely visible)
Vysor quickly flows through a few text strings that I can't read but settles on "Installing Vysor APK..." and just sits there
So far, Vysor has not actually error'd out, just hangs.  
If I exit back to the main screen and try to reconnect, Vysor hangs on "Connecting..."

**Expected behavior**
Screen Mirror

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Chrome Version 92.0.4515.111 (Official Build) beta (64-bit)

**Android (please complete the following information):**
 - Device: Samsung Galaxy S8
 - OS: Android (unable to locate version due to low screen visibility, but it is up to date)
 - Vysor Bug Report Link: https://gist.github.com/cb436db4f0f7cfca3493d4795bb32537
 - Hangs on "Connecting..." screen after exiting mirroring tab: https://gist.github.com/3628a27c2277e222380d93f1dadc8067

**Additional context**
As noted above, the phone screen is barely visible due to damage.  Prior to damage, USB debugging was enabled.  I attempted to install the APK on the phone via https://app.vysor.io/Vysor-release.apk (found in another bug report) but nothing seems to download.  When Vysor hangs on installing APK screen, the phone does not seem to indicate that anything is actually happening.  Hanging on the APK screen happens in both the web version and the desktop version of Vysor.

**Comment by koush** [comment]
I'd try to use the native app vs the chrome app. Sorry, I don't provide support for damaged phones.

**Comment by SPENCRY** [comment]
Using the native app gets me the same hangup point.

I have the screen back working sufficiently to see everything.  Besides the screen issues, the phone works fine otherwise.

I have made sure USB debugging is on.  USB mode is file transfer.  USB debugging is allowed once Vysor is connected.  

I installed the Vysor app from the play store but it appears it has been removed.

Any other recommendations?
